### DISCUSSION TITLE
Failed to set permissions during startup

### DISCUSSION THREAD

**Discussion by Stu619** [ltsp/ltsp#402] [Q&A]
On starting an LTSP client it seems to get most of the way through but then has a number of errors regarding "Failed to set Permissions" on a bunch of hidden files in /etc/ltsp the files mentioned don't exist in that folder.

**Comment by alkisg** [comment]
Are you talking about this issue of Ubuntu 20.10?
https://github.com/ltsp/ltsp/discussions/388

**Reply by Stu619** [reply]
I don't think so. The error says "Failed to set permissions on /etc/ltsp/.epoptes.crt.xxx...operation not supported". that .epoptes file doesn't exist on my system. It then does the same thing for several other hidden files in /etc/ltsp.

**Comment by alkisg** [comment]
Can you upload a photo of that message?
Does it break booting, does it stop there?

**Comment by Stu619** [comment]
Here is a photo of the error message.

**Comment by alkisg** [comment]
Where?

If you sent it as an email attachment, I'm not sure that it reached github, see your post there:
https://github.com/ltsp/ltsp/discussions/402#discussioncomment-366905

**Reply by Stu619** [reply]
Here is the photo of the error message.

![error-message](https://user-images.githubusercontent.com/78970108/107884121-fd74d280-6ec0-11eb-9414-8b119b5b05f8.jpg)

**Comment by alkisg** [comment]
This is indeed the rsync error that I linked to; either switch to another rsync version, or use another Ubuntu version.

**Reply by Stu619** [reply]
Thanks a lot. Would Ubuntu 20.04 work?

**Comment by alkisg** [comment]
Sure, all other Ubuntu versions don't have this bug.
You could also download and install the older deb from here:
https://packages.ubuntu.com/focal/rsync

```shell
wget http://gr.archive.ubuntu.com/ubuntu/pool/main/r/rsync/rsync_3.1.3-8_amd64.deb
dpkg -i rsync_3.1.3-8_amd64.deb
apt-mark hold rsync
ltsp image /
```

**Reply by Stu619** [reply]
Thanks, that worked for me and my client machine gets to the login page but now it will not allow any users to login.

**Comment by alkisg** [comment]
Did you run `ltsp initrd` after adding user accounts on the server?
On a client press Ctrl+Alt+F2 to switch to vt2, and try to login there. If it fails, what error message does it show?

**Reply by Stu619** [reply]
I ran that. Both the graphical login and the command line login just say "login incorrect". It's the right username and password for sure and that user is not logged in anywhere else. I did notice when the client was booting a note about an SSSD dependency problem. Could that also be an Ubuntu 20.10 issue?

**Comment by alkisg** [comment]
In normal Ubuntu 20.10 installations, SSSD isn't installed and there are no messages about it.
If you're trying to use SSSD, you need to mention it for the troubleshooting process...

Otherwise, read the last paragraph of this page: https://ltsp.org/man/ltsp.conf/
It will tell you how to log in as root in clients. So, log in as root on a client, then type `grep username /etc/passwd /etc/shadow` and paste the results here. Then try `su - username` and see if that one works.

P.S. you may also come to IRC for remote support if you like; if so, go to ltsp.org and click on the IRC link.

**Comment by Stu619** [comment]
I decided to change to Ubuntu 20.04 and everything worked perfectly right from the start. Thanks for all your help with these issues.
